### PR TITLE
Fix/167971554

### PR DESCRIPTION
**Pull request by ricardoul**

### CODE REVIEW

File: src/components/ActivityFeed/ActivityFeed.js [hunk 1]
```diff
@@ -228,7 +228,13 @@ class ActivityFeed extends React.Component<Props, State> {
         decimals = 18,
         iconUrl,
       } = assets.find(({ symbol }) => symbol === notification.asset) || {};
-      const value = utils.formatUnits(new BigNumber(notification.value.toString()).toFixed(), decimals);
+      let bigNumber;
```
Comment (jsidorenko) [inline src/components/ActivityFeed/ActivityFeed.js]: that variable name means nothing

File: src/components/ActivityFeed/ActivityFeed.js [hunk 2]
```diff
@@ -228,7 +228,13 @@ class ActivityFeed extends React.Component<Props, State> {
...
+      try {
+        bigNumber = new BigNumber(notification.value.toString());
```
Comment (jsidorenko) [inline src/components/ActivityFeed/ActivityFeed.js]: `bigNumber = new BigNumber(get(notification, 'value', 0).toString());`
and no need for try..catch

File: src/components/ActivityFeed/ActivityFeed.js [hunk 3]
```diff
@@ -441,6 +447,7 @@ class ActivityFeed extends React.Component<Props, State> {
           stickySectionHeadersEnabled={false}
         />
 
+        {selectedEventData &&
```
Comment (jsidorenko) [inline src/components/ActivityFeed/ActivityFeed.js]: should be `{!!selectedEventData &&`

File: src/components/ActivityFeed/ActivityFeed.js (hunk 2 above)
Comment (grilix) [reply src/components/ActivityFeed/ActivityFeed.js]: That would handle the `undefined` scenario, right? What do you think about the decimals? Is that something we need to take care of?

Comment (jsidorenko) [reply src/components/ActivityFeed/ActivityFeed.js]: I wouldn't care about decimals here

**Comment by grilix** [comment]
This was fixed on another PR. Extracted the tests to #1341

**Pull request closed without merging**